### ISSUE TITLE
revocation document / revocationList should be linked data

### ISSUE THREAD

**Issue by ottonomy** [openbadges/openbadges-specification#33]
Currently the spec instructs issuers to deliver `{"revoked": true}` as the whole body of a request for a revoked badge. Given that the other hosted components of badges are linked data, this should be linked data as well, but that'll require adding a couple more terms to the context. Currently, the revocation reason is only a feature of signed badges revocationLists, though there's no reason for that to apply to signed badges but not hosted badges.

I expect something like the following would be a more useful document when a consumer is expecting to see an Assertion (so they're expecting linked data):

``` json
{
  "@context": "https://w3id.org/openbadges/v1",
  "revoked": true,
  "revocationReason": "Code of conduct violation"
}
```

**Comment by ottonomy** [comment]
* As a developer of a badge verification service, I would like revocation list documents to be delivered as linked data like the rest of the Open Badges data classes.

**Comment by ottonomy** [comment]
Prototype of recovationList:
```jsonld
{
  "@context": "https://w3id.org/openbadges/v2",
  "@vocab": "https://w3id.org/openbadges#uid",
  "qp8g1s": "Issued in error",
  "2i9016k": "Issued in error",
  "1av09le": "Honor code violation"
}
```
RevocationLists are currently not in a style amenable to linked data processing, because the key values are not required to be IRIs, and may be any string. The above example uses the `@vocab` sledgehammer to force them to become IRIs.

I've never been a fan of revocationLists. I would rather see them disappear and be replaced in assertions by verification suites that describe possible methods of checking signed badges against revocation, included a hosted URL check, but also capable of handling emergent systems using blockchains or other distributed storage systems. Some interesting work is being done with [ChainPoint](http://www.chainpoint.org/) and [Linked Data Signatures](https://w3c-dvcg.github.io/ld-signatures/#signature-suites)

**Comment by mgylling** [comment]
Closing this issue, as it was resolved by the 2.0 Recommendation. This issue and all other issues resolved by 2.0 can be found via the 2.0 Release Candidate milestone and PR #99.
